WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/MySitesPage.java

```diff
@@ -111,6 +111,7 @@ public MySitesPage switchToSite(String siteUrl) {
         return this;
     }
 
+    // Either this or goToStats should be removed after making the implementation uniform.
```

Comment:
❓   Is there more coming for this PR to handle this comment; as I see both methods still hanging around? And yes, I did notice that `clickStats` is also being called from within `StatsTest`.

Reply:
I can address the issue in this PR once I'm resuming work on Monday 🙇🏻.
At the time of opening the PR and afterwards It was a bit difficult to find time for it.

Reply:
Working on it 🚀

Reply:
Done. While on it I also had to fix (again 😅 ) the navigation to a some site menu items since it was failing now that for an unclear reason the quick actions are visible again.

I adapted the navigation methods so they work both when quick actions is present and when it's not.

Reply:
> I am still seeing the grey background, but I know that this will be addressed separately (after this PR is merged into the fix/screenshots-test branch).

@zwarm Will look into it once we're done merging this into `fix/screenshots-test` 🙇 .